cloudant-sync-datastore-core/src/test/java/com/cloudant/sync/internal/documentstore/DatabaseImplRevsDiffTest.java

```diff
@@ -26,7 +26,7 @@
 public class DatabaseImplRevsDiffTest extends BasicDatastoreTestBase{
 
     @Test
-    public void revsDiff_emptyInput_returnEmpty() {
+    public void revsDiff_emptyInput_returnEmpty() throws Exception {
```

Comment:
Does this need to be `Exception` or can `DocumentStoreException` meet the requirement?